fedora-pr-review-guide.md

```diff
@@ -93,6 +94,10 @@ Copy & paste to your PR review.
         When multiple patches are mereged together and you need to list more authors,
         you can add `Co-Authored-By: John Doe <john@doe.com>` to the commit message (multiple times).
       - Link to the upstream commits/PRs the patch is derived from.
+  - If you create patches from scratch (e.g. to fix an issue)
+    - Patches should ideally be proposed upstream (and a link to the PR should be in the patch-related comment).
```

Comment:
Lat minute change, sorry about that.

```suggestion
    - Patches should ideally be proposed upstream (and a link to the PR should be in the spec).
```

This is easier and allows using the PR URL as the Patch definition.

Reply:
+1